src/coreclr/hosts/unixcoreruncommon/coreruncommon.cpp

```diff
@@ -16,7 +16,11 @@
 #include <string>
 #include <string.h>
 #include <sys/stat.h>
-#ifdef HAVE_SYS_SYSCTL_H
```

Comment:
Sounds ok for now. Thanks for tracking down the culprit. While I would like to eventually fix the test, I'd like to make sure it fixes the problem on the Alpine Linux and others that caused the addition of the HAVE_SYS_SYSCTL_H. I'll create an issue for it.